plotly/figure_factory/_facet_grid.py

```diff
@@ -25,6 +25,7 @@
 THRES_FOR_FLIPPED_FACET_TITLES = 10
 GRID_WIDTH = 1
 
+VALID_TRACE_TYPES = ['scatter', 'scattergl', 'histogram', 'bar', 'box']
```

Comment:
Since `marker` is not a valid key in facet grid, I vote that these trace types get moved into another PR/Issue. It would require a large reworking of the code, so I think we should move it to another issue so we can get this merged so the documentation can go up this week.